src/orquestra/sdk/_base/cli/_dorq/_workflow/_results.py

```diff
@@ -55,7 +57,7 @@ def on_cmd_call(
                 config=config,
             )
         except Exception as e:
-            self._presenter.show_error(e)
+            self._error_presenter.show_error(e)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Do you have a code path to test this?

Reply:
Good point. Added!